src/building-blocks/FieldSet.tsx

```diff
@@ -25,11 +25,13 @@ export default class FieldSet extends Component<IProps> {
       ;
 
     return (
-      <Antd.Row {...rowProps} className={cx(className)}>
-        <Legend fieldSet={fieldSet} />
+      <div className={cx('fields-ant-field-set', className)}>
```

Comment:
I'm not sure what the purpose of a wrapper div would be here. Any reason in particular?

Reply:
Just styling, so you can add horizontal borders without messing with the styling / margins on Rows. This is what we use now, and from what I can tell it wouldn't be wise to do the bordering on the Row component.

Reply:
Could be removed if we find out otherwise after style changes. Doesn't make a difference though.

Reply:
Sounds good

Reply:
I would recommend having a const `CLASS_PREFIX = 'fields-ant'` or something similar for when we add classes to other components inevitability